lib/converters/v1.0.0/converter-v1-to-v2.js

```diff
@@ -178,8 +178,10 @@ _.assign(Builders.prototype, {
         // If the query params have been altered, update the raw stringified URL
         queryParamAltered && (parsed.raw = util.urlunparse(parsed));
 
-        // return the objectified URL only if query param or path variable descriptions are present, string otherwise
-        return (parsed.query || parsed.variable) ? parsed : (parsed.raw || requestV1.url);
+        // return the objectified URL only if query param or path variable descriptions are present,
+        // otherwise empty string if requestV1.url has null value, else the string
+        return (parsed.query || parsed.variable) ? parsed :
+            (parsed.raw || (_.isNull(requestV1.url) ? '' : requestV1.url));
```

Comment:
We should also handle the case for `undefined`. This can simply be: 
```suggestion
            (parsed.raw || requestV1.url || '');
```